airflow/providers/google/cloud/operators/vertex_ai/auto_ml.py

```diff
@@ -170,6 +172,8 @@ def __init__(
         self.quantiles = quantiles
         self.validation_options = validation_options
         self.budget_milli_node_hours = budget_milli_node_hours
+        self.window_stride_length = window_stride_length,
+        self.window_max_count = window_max_count,
```

Comment:
```suggestion
        self.window_stride_length = window_stride_length
        self.window_max_count = window_max_count
```

In this case comma in the end transform into the tuples `(window_stride_length, )` and `(window_max_count, )` 
I guess it added accidentally

Reply:
And also suggested changes will fix the tests